doc/source/report/distribution.rst

```diff
@@ -74,20 +131,22 @@ of the tail.
 
 Here are our algorithms:
 
-- Draw the histogram for our data.  Observe the distribution and intuitively
-  figure out whether our distribution assumption makes sense.
-- Estimate parameters based on MoM or MLE.
-- Add the density function to our histogram, see the fitness of
-  our distribution.
-- Conduct statistical test to quantitatively analysis the fitness (Pearson chi
-  square test). For testing the hypothetical distributions of a given array,
-  there are several existing commonly used methods. However, each approach has
-  their pros and cons. Following is a short overview of these testing framework.
-  We recommend that all the methods are to be tried to get a comprehensive
-  understanding of the inter-event step distributions.
-- Pearson Chi-square test
-- Fisher’s exact test
-- KS test
+- Draw the histogram for our data. For each mouse day,  observe the distribution and explore whether our distribution assumption makes sense.
+- Estimate parameters based on MLE of powerlaw and exponential.
```

Comment:
MLE of "truncated" power law and "truncated" exponential. Our model is truncated, not the full model. Also, the truncated power law distribution has another name called Pareto distribution.